build/package.yml

```diff
@@ -26,11 +26,11 @@ steps:
     inputs:
       command: pack
       configuration: '$(buildConfiguration)'
+      packagesToPack: '**/*.csproj;!test/**/*.csproj;!**/*.UnitTests.csproj'
```

Comment:
I remove the packing of test projects, and have removed the web tests and integration tests from the `nuget` artifact. The pipeline instead uses these projects from the `IntegrationTests` artifact, as they are built in a previous task.